translator/translate/otel/exporter/awsxray/translator.go

```diff
@@ -102,6 +103,9 @@ func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 	if resourceARN, ok := common.GetString(conf, common.ConfigKey(common.TracesKey, resourceARNKey)); ok {
 		cfg.AWSSessionSettings.ResourceARN = resourceARN
 	}
+	if transitOtlp, ok := common.GetBool(conf, common.ConfigKey(common.TracesKey, transitSpansInOtlpFormatKey)); ok {
+		cfg.TransitSpansInOtlpFormat = transitOtlp
```

Comment:
can we include this as part of our test in our translation test?

add the necessary json config here: https://github.com/aws/amazon-cloudwatch-agent/blob/main/translator/tocwconfig/sampleConfig/trace_config.json

Then add the yaml config here: https://github.com/aws/amazon-cloudwatch-agent/blob/main/translator/tocwconfig/sampleConfig/trace_config_linux.yaml#L2

Reply:
thanks, i have updated this PR to cover that.